libs/build_met.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash                                                                                                                              
+
+set -eux
+
+name="met"
+version=${1:-${STACK_met_version}}
+release_date=${2:-${STACK_met_release_date}}
+
+cd  ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+software=$name-$version.$release_date
+pkg_name=$name-$version
+url="https://github.com/dtcenter/MET/releases/download/v$version/$software.tar.gz"
+[[ -d $software ]] || ( $WGET $url; tar -xf $software.tar.gz )
+[[ -d $pkg_name ]] && cd $pkg_name || ( echo "$pkg_name does not exist, ABORT!"; exit 1 )
```

Comment:
please follow the structure of other build scripts.
This section belongs after the modules and prefixes are set.
Also, this script is missing an option for `DOWNLOAD_ONLY`

Reply:
Done.

Reply:
@ aerorahul This is not actually done.  I will revisit this and finish adding this necessary section.  I did not realize how it worked until looking at a later comment.  I will follow up with any questions.

Reply:
Actually this is done (not pushed yet).  I looked into it further and now better understand how it is used.